layers/sflow.go

```diff
@@ -1832,6 +1832,59 @@ func decodeExtendedUserFlow(data *[]byte) (SFlowExtendedUserFlow, error) {
 	return eu, nil
 }
 
+// **************************************************
+//  Packet IP version 4 Data Record
+// **************************************************
+
+//  0                      15                      31
+//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
+//  |      20 bit Interprise (0)     |12 bit format |
+//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
+//  |                  record length                |
+//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
+//  /           Packet IP version 4 Record          /
+//  /                                               /
+//  +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
+
+type SFlowIpv4DataRecord struct {
```

Comment:
Can we modify the existing SFlowIpv4Record directly? Is there any concern not to touch the original code?

Reply:
Ack